### DISCUSSION TITLE
ERROR "Among us directory was not found, is it installed? Try running it  once"

### DISCUSSION THREAD

**Discussion by paulopimentel** [Impostor/Impostor#289] [Q&A]
When I click save to start the client, the following message "Among us directory was not found, is it installed? Try running it  once" how to solve?

**Comment by fancythedeveloper** [comment]
It literally says what's the issue.

**Comment by Cloudy71** [comment]
Close, the issue was solved on Discord.

**Comment by Fluffy728** [comment]
“ I believe this refers to the old Patcher software, which is no longer supported to set up custom servers. Please use the website instead: https://impostor.github.io/Impostor/ “

From miniduikboot at https://github.com/Impostor/Impostor/issues/575
